sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -129,7 +129,16 @@ case class DescribeFunction(
             result
           }
 
-        case None => Seq(Row(s"Function: $functionName not found."))
+        case None =>
+          val currentDb = sparkSession.sessionState.catalog.getCurrentDatabase
+          if (sparkSession.externalCatalog.functionExists(currentDb, functionName)) {
+            val catalogFunction = sparkSession.externalCatalog.getFunction(currentDb, functionName)
+            Row(s"Function: ${catalogFunction.identifier}") ::
+              Row(s"Class: ${catalogFunction.className}") ::
+              Row(s"Usage: N/A") :: Nil
+          } else {
+            Seq(Row(s"Function: $functionName not found."))
+          }
```

Comment:
Looks like we only need to change https://github.com/apache/spark/pull/12885/files#diff-d2a203f08c862bd762e6740c16e972f7R118 to `sparkSession.sessionState.catalog.lookupFunctionInfo`?